survey/survey-service-offering_instance.json

```diff
@@ -0,0 +1,86 @@
+{
+  "@context": {
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "survey": "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/survey/",
+    "gx": "https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#",
+    "general": "https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/"
+  },
+  "@id": "did:example:survey_service_offering_0xy5vhw3heyjqb4d347hc9zrkw",
+  "@type": "survey:SurveyServiceOffering",
+  "survey:surveyUrl": {
+    "@value": "https://web-mpc.gxfs.gx4fm.org/?session=0xy5vhw3heyjqb4d347hc9zrkw",
+    "@type": "xsd:string"
+  },
+  "survey:surveyCreationTime": {
+    "@value": "2023-08-01T15:14:04",
+    "@type": "xsd:dateTime"
+  },
+  "survey:surveyEndTime": {
+    "@value": "2023-08-02T17:14:00",
+    "@type": "xsd:dateTime"
+  },
+  "gx:description": {
+    "@value": "End to end supply chain survey on important leading and lagging indicators for semiconductor supply chains such as demand & sales development and inventory metrics to forecast market development",
+    "@type": "xsd:string"
+  },
+  "gx:name": {
+    "@value": "Semiconductor True Demand Survey",
+    "@type": "xsd:string"
+  },
+  "survey:general": {
+    "@id": "did:web:registry.gaia-x.eu:General:PlFjHxuQoH0v7iaVC0QqdxmtQyqY-LQX6D-p",
+    "@type": "general:General",
+    "general:description": {
+      "@type": "general:Description",
+      "gx:name": {
+        "@value": "Semiconductor True Demand Survey",
+        "@type": "xsd:string"
+      },
+      "gx:description": {
+        "@value": "End to end supply chain survey on important leading and lagging indicators for semiconductor supply chains such as demand & sales development and inventory metrics to forecast market development",
+        "@type": "xsd:string"
+      }
+    },
+    "general:data": {
+      "@type": "general:Data",
+      "general:size": {
+        "@value": "5.8",
+        "@type": "xsd:float"
+      },
+      "general:contractId": {
+        "@value": "contract_zcdkr7kqd47y0w5b4tg91w1etw",
+        "@type": "xsd:string"
+      }
+    },
+    "general:links": {
+      "@type": "general:Links",
+      "general:data": {
+        "@type": "general:Link",
+        "general:url": {
+          "@value": "https://edcdb-pr.gxfs.gx4fm.org/",
+          "@type": "xsd:anyURI"
+        },
+        "general:type": "Video"
+      }
+    },
+    "general:bundleData": {
+      "@type": "general:BundleData",
+      "general:requiredData": {
+        "@type": "general:Link",
+        "general:url": {
+          "@value": "https://web-mpc.gxfs.gx4fm.org/?session=0xy5vhw3heyjqb4d347hc9zrkw",
+          "@type": "xsd:anyURI"
+        },
+        "general:type": "3DPreview"
+      },
+      "general:relatedData": {
+        "@type": "general:Link",
+        "general:url": {
+          "@value": "https://web-mpc.gxfs.gx4fm.org/?session=0xy5vhw3heyjqb4d347hc9zrkw",
```

Comment:
siehe result offering instance

Reply:
s.o.

Reply:
s.o.

Reply:
changed type of general:type to a more fitting enum, removed bundleData